static/templates/settings/display_settings.hbs

```diff
@@ -111,7 +111,7 @@
     <div class="advanced-settings {{#if for_realm_settings}}settings-subsection-parent{{else}}subsection-parent{{/if}}">
         <div class="subsection-header">
             <h3 class="light">{{t "Advanced" }}</h3>
-            {{> settings_save_discard_widget section_name="advanced-settings" show_only_indicator=(not for_realm_settings) }}
+            {{> settings_save_discard_widget id_prefix="org" section_name="advanced-settings" show_only_indicator=(not for_realm_settings) }}
```

Comment:
Should the `id_prefix` depend on `for_realm_settings` on this page? 

Relatedly I notice `static/templates/settings/organization_profile_admin.hbs` seems to not have `id_prefix` changed in this commit.

Reply:
Fixed in #23780.